color-picker-project/src/components/colorpicker/ColorPicker.css

```diff
@@ -39,4 +40,28 @@
 
 .color-picker-alpha {
   margin-top: 1rem;
+}
+
+@media (max-width: 1080px) {
+    .color-picker-title {
+        font-size: 2.5rem;
+    }
+    .color-picker-color {
+        width: 23rem;
+    }
+}
+
+@media (min-width: 1080px) {
+    .color-picker-color {
+        width: 70%;
+    }
+}
+
+@media (max-width: 800px) {
+    .color-picker-title {
+        font-size: 2rem;
+    }
+    .color-picker-color {
```

Comment:
This is already being set in the max-width: 1080px media query, it can be deleted here.

Reply:
This is a smaller font size for max-width 800px. Keeping this.

Reply:
It probably would have been more clear if I'd placed this comment 1 line lower.  I was referring to the .color-picker-color width of 23rem which is also set to 23rem in the max-width 1080px media query. Yeah, the font size is definitely good in this media query!